envs/monkey_zoo/terraform/firewalls.tf

```diff
@@ -323,9 +323,9 @@ resource "google_compute_firewall" "deny-rdp64-rdp65-to-others" {
   source_tags = ["rdp-64", "rdp-65"]
 }
 
-// We are disabling PowerShell because we want only RDP to run on these machines
+// We are disabling PowerShell because we want only RDP\SMB to run on these machines
 // and we can't do it via Packer because it uses WinRM to configure the instances
```

Comment:
? We can't run a powershell command using packer? GCP Firewall rules are meant to describe the communication between networks, not disabling services on specific machines.

Reply:
Maybe we can write a different issue about that and leave this as workaround.